src/Tools/Source/RunTests/Options.cs

```diff
@@ -185,7 +185,7 @@ internal class Options
                 Console.WriteLine($"Did not find artifacts directory at {artifactsPath}");
                 return null;
             }
-            resultFileDirectory ??= Path.Combine(artifactsPath, "TestResults");
+            resultFileDirectory ??= Path.Combine(artifactsPath, configuration, "TestResults");
```

Comment:
Should the configuration come last?

```suggestion
            resultFileDirectory ??= Path.Combine(artifactsPath, "TestResults", configuration);
```